### ISSUE TITLE
Lomiri: session indicator cannot be launched via systemctl, but manually

### ISSUE THREAD

**Issue by sunweaver** [AyatanaIndicators/ayatana-indicator-session#76]
When using the session indicator in Lomiri, the indicator fails to start (at least this can be observed in the UBports PDK).

However, when launching the session indicator service from a terminal (PDK serial terminal), the indicator comes up.

This is what gets reported by journalctl:

```
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Main process exited, code=killed, status=5/TRAP
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Failed with result 'signal'.
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Scheduled restart job, restart counter is at 1.
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Main process exited, code=killed, status=5/TRAP
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Failed with result 'signal'.
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Scheduled restart job, restart counter is at 2.
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Main process exited, code=killed, status=5/TRAP
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Failed with result 'signal'.
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Scheduled restart job, restart counter is at 3.
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Main process exited, code=killed, status=5/TRAP
Jun 19 13:12:52 linux systemd[704]: ayatana-indicator-session.service: Failed with result 'signal'.
Jun 19 13:12:53 linux systemd[704]: ayatana-indicator-session.service: Scheduled restart job, restart counter is at 4.
Jun 19 13:12:53 linux systemd[704]: ayatana-indicator-session.service: Main process exited, code=killed, status=5/TRAP
Jun 19 13:12:53 linux systemd[704]: ayatana-indicator-session.service: Failed with result 'signal'.
Jun 19 13:12:53 linux systemd[704]: ayatana-indicator-session.service: Scheduled restart job, restart counter is at 5.
Jun 19 13:12:53 linux systemd[704]: ayatana-indicator-session.service: Start request repeated too quickly.
Jun 19 13:12:53 linux systemd[704]: ayatana-indicator-session.service: Failed with result 'signal'.
```

**Comment by peat-psuwit** [comment]
On my Xperia X, a little more gets reported:

```
Jun 23 01:15:55 ubuntu-phablet systemd[2333]: Started Ayatana Indicator Session Service.
Jun 23 01:15:55 ubuntu-phablet ayatana-indicat[9908]: Settings schema 'org.gnome.settings-daemon.plugins.media-keys' is not installed
Jun 23 01:15:55 ubuntu-phablet systemd[2333]: ayatana-indicator-session.service: Main process exited, code=killed, status=5/TRAP
Jun 23 01:15:55 ubuntu-phablet systemd[2333]: ayatana-indicator-session.service: Failed with result 'signal'.
Jun 23 01:15:55 ubuntu-phablet systemd[2333]: ayatana-indicator-session.service: Scheduled restart job, restart counter is at 1.
Jun 23 01:15:55 ubuntu-phablet systemd[2333]: Stopped Ayatana Indicator Session Service.
```

That means this would be fixed by #72, which is unreleased yet, and depends on the MR at lomiri-schemas too [1].

The reason I haven't merged [1] at first glance is that I'm not sure if it's a good idea to move. In the Canonical's world they don't care about the current DE at all, and thus Unity 8 has historically been using Gnome's schema. 

But since #72 has been merged anyway I'll consider merging [1].

But just in case, 

[1] https://gitlab.com/ubports/core/lomiri-schemas/-/merge_requests/10

**Comment by peat-psuwit** [comment]
Heck, Lomiri itself doesn't even read the Gnome's GSettings.

https://gitlab.com/ubports/development/core/lomiri/-/blob/main/qml/Components/Dialogs.qml#L122

But should it read some GSettings though?

**Comment by peat-psuwit** [comment]
So, the settings seem to eventually be used to set GMenuItem's `"accel"` attribute.

https://github.com/tari01/ayatana-indicator-session/blob/04290777475d93c06c412cf4156a34caf340ca1e/src/service.c#L682

The question is, what interpret this? And if the answer is Lomiri itself, maybe we can just remove the support for this schema altogether.

**Comment by peat-psuwit** [comment]
Fixed by #78.
